nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -203,7 +207,9 @@ def __init__(
             self._feat_out = d_model
 
     @typecheck()
-    def forward(self, audio_signal, length):
+    def forward(self, audio_signal, length=None):
+        if length is None:
```

Comment:
Do we ever expect length to be None?

Reply:
Actually it can be None for Jasper/QuartzNet as they do not use this in the current implementation. That is why the export method was working without passing the length. Take a look here: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/asr/modules/conv_asr.py#L189 . We have None as default.

If user does not pass the length, we would not mask-out anything assuming all inputs are speech. It can be just one sample or a batch. It is a optional feature. With the fix in this PR on the export, I can drop it, but good to have I think.

Reply:
Such an interesting issue.. Cool fix. Yes, lets keep it.